cmd/flow-rosetta-server/README.md

```diff
@@ -33,12 +33,13 @@ rosetta-cli check:data --configuration-file flow.json
 
 ```sh
 Usage of flow-rosetta-server:
-  -a, --api string              host URL for GRPC API endpoint (default "127.0.0.1:5005")
-  -e, --cache uint              maximum cache size for register reads in bytes (default 1073741824)
+  -a, --address string          address to host Rosetta API on (default "127.0.0.1:8080")
+  -d, --dps string              host URL for DPS API endpoint (default "127.0.0.1:5005")
+  -f, --flow string             host address for Flow network's Access API endpoint
   -l, --level string            log output level (default "info")
-  -p, --port uint16             port to host Rosetta API on (default 8080)
-  -t, --transaction-limit int   maximum amount of transactions to include in a block response (default 200)
       --smart-status-codes      enable smart non-500 HTTP status codes for Rosetta API errors
+      --cache-size uint         maximum cache size for register reads in bytes (default 1000000000)
```

Comment:
wdyt of adding underscores in there to make this more readable?

```suggestion
      --cache-size uint         maximum cache size for register reads in bytes (default 1_000_000_000)
```

Reply:
I had it like that before, but it won't show up in the help for the command anyway.